linkstone-shim/src/main/java/net/minecraft/server/MojangStatisticsGenerator.java

```diff
@@ -0,0 +1,65 @@
+package net.minecraft.server;
+
+import net.glowstone.linkstone.annotations.LClassfile;
+import net.glowstone.linkstone.annotations.LField;
+import net.glowstone.linkstone.annotations.LGenerate;
+import net.glowstone.linkstone.annotations.LMethod;
+
+import static net.glowstone.linkstone.annotations.Version.V1_12_R1;
+
+@LClassfile(version = V1_12_R1)
+public class MojangStatisticsGenerator {
+    @LGenerate
+    @LField(version = V1_12_R1, name = "e")
+    private final IMojangStatistics iMojangStatistics;
+
+    @LGenerate
+    @LField(version = V1_12_R1, name = "h")
+    private final long startTime;
+
+    //TODO @Isaiah Deobfuscate and Annotated Method
+    private boolean i;
```

Comment:
Guessing isRunning. Changes to true after first call to a()

Reply:
Is that right 9c3ba02d1dc2bd08f07e515bb6c61fb242846b72?

Reply:
Yes